Victoria & Albert Museum.js

```diff
@@ -0,0 +1,198 @@
+{
+	"translatorID": "56fc13bf-411f-4409-9a61-3d334d76763b",
+	"label": "Victoria & Albert Museum",
+	"creator": "V&A Digital Media & Publishing",
+	"target": "^https?://collections\\.vam\\.ac\\.uk",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2021-02-12 14:45:24"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2020 V&A Museum
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.includes("/item") != -1) {
```

Comment:
Should just be `if (url.includes("/item")) {` - `includes` returns a boolean.

Reply:
Also make this `"/item/"`, because the regex match at the top of `scrape` will throw an error if there's no trailing slash in the URL. Not sure that can ever happen, but consistency is good.